src/components/GoToTopButton.tsx

```diff
@@ -0,0 +1,23 @@
+import { memo } from "react"
+import { IoChevronUpSharp } from "react-icons/io5"
+
+import { Button } from "./Buttons"
+
+const scrollToTop = () => {
+  window.scrollTo({ top: 0, behavior: "smooth" })
+}
+
+export const GoToTopButton = memo(() => {
```

Comment:
@alex1092 `memo` is not needed for this component

Reply:
Yep no worries, there had been design changes so I need to refactor this again